<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inHis</class>
 <widget class="QDialog" name="inHis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>입력</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 245, 221);</string>
  </property>
  <widget class="QComboBox" name="inCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 245, 158);</string>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
   <property name="maxCount">
    <number>7</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>월급</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>용돈</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>기타</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Hvalue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 245, 158);</string>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="outCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 245, 158);</string>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
   <property name="maxCount">
    <number>7</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>공과금</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>교통비</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>생필품</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>식비</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>의류,미용</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>의료비</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>기타</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="hName_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Hvalue_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 230, 130);</string>
   </property>
   <property name="text">
    <string> 수입</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 230, 130);</string>
   </property>
   <property name="text">
    <string> 지출</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
